bokehjs/src/coffee/models/widgets/tables/data_table.coffee

```diff
@@ -230,3 +231,12 @@ export class DataTable extends TableWidget
   @internal {
     default_width:        [ p.Number, 600   ]
   }
+
+  get_scroll_index: (grid_range, selected_indices) ->
+    if not @scroll_to_selection or selected_indices.length == 0
+      return null
+
+    if not any(selected_indices, (i) -> grid_range.top <= i <= grid_range.bottom)
+      return Math.max(0, Math.min.apply(null, selected_indices) - 1)
```

Comment:
This could be simplified to `Math.min(selected_points...)` in CS and `Math.min(...selected_points)` in TS.